src/views/minor/Footer.vue

```diff
@@ -1,20 +1,33 @@
 <template>
     <footer class="page-footer font-small">
-        <div class="footer-copyright text-center py-3" style="color: white">
+        <div class="text-center py-3" style="color: white">
             {{ copyright }}
+            <p>
+                Licensed under the
+                <a
+                    class="footer-link"
+                    href="https://opensource.org/licenses/MIT">
+                    MIT License</a
+                >.
+            </p>
+            <p>
+                By using this website, you agree any images you create are
+                released into the public domain.
```

Comment:
Can we reword this as "By using this website, you agree that any images or media you create using the tools on this website are released by you into the public domain according to the [CC0 license](https://creativecommons.org/publicdomain/zero/1.0/)."

Reply:
Should be fixed in https://github.com/numberscope/frontscope/commit/53aa6b1ae8f0d7baa5bddcdba7c69a9fb09fd8a9. I omitted the word "license" as I wasn't sure if there's a legal distinction between a public domain "dedication" and a "license".